From ab7f4dd875ded54e8fa3e6212c2c35c7848d2779 Mon Sep 17 00:00:00 2001
From: "Badrappan, Jeevaka" <jeevaka.badrappan@intel.com>
Date: Thu, 17 Aug 2023 09:46:37 +0530
Subject: [PATCH] Add libtflite_static to address undefined symbol build error

Build of android.hardware.neuralnetworks@1.3-generic-impl fails with
undefined symbol tflite::CpuBackendContext::PreferGemmlowpOnX86 error.

PreferGemmlowpOnX86 is defined in tflite_static and it is not added
as part of static library list.

Fixed the issue by adding tflite_static to static_libs list.

Tracked-On: OAM-110093
Signed-off-by: Badrappan, Jeevaka <jeevaka.badrappan@intel.com>
---
 Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.bp b/Android.bp
index c4d8070..18771d2 100644
--- a/Android.bp
+++ b/Android.bp
@@ -107,6 +107,7 @@ cc_library_shared {
         "libpugixml",
         "libneuralnetworks_common",
         "libngraph_creator",
+        "libtflite_static",
     ],
 
     generated_headers: [
-- 
2.40.0

